src/settings-personal.js

```diff
@@ -24,15 +24,11 @@ import Vuex from 'vuex'
 import UserSettings from './views/UserSettings.vue'
 import DailySummary from './views/DailySummary.vue'
 import { translate as t, translatePlural as n } from '@nextcloud/l10n'
-import { generateFilePath } from '@nextcloud/router'
 import store from './settings-store.js'
 
 Vue.prototype.t = t
 Vue.prototype.n = n
 
-// eslint-disable-next-line no-undef, camelcase
-__webpack_public_path__ = generateFilePath(appName, '', 'js/')
```

Comment:
This is not necessary anymore as we use vite, right ?

Reply:
Right this now done with `renderBuiltUrl` in the viteconfig